README.md

```diff
@@ -31,51 +32,99 @@ Imagine a declarative Kubernetes-style API for _anything_, supported by an ecosy
+`kcp` could be _useful_ for local development scenarios, where you don't necessarily care about all of Kubernetes' many built-in resources and their reconciling controllers.
 
-#### Multi-Tenant Kubernetes?
+### Embedded/low-resource scenarios?
 
-`kcp` could be _useful_ for multi-tenancy scenarios, by allowing [multiple tenant clusters inside a cluster](docs/investigations/logical-clusters.md) to be managed by a single `kcp` control plane.
+`kcp` could be _useful_ for environments where resources are scarce, by limiting the number of controllers that need to run. Kubernetes' asynchronous reconciliation pattern can also be very powerful in disconnected or intermittently connected environments, regardless of how workloads actually run.
 
-#### Local Kubernetes Development?
+## What about a long-term vision?
 
-`kcp` could be _useful_ for local development scenarios, where you don't necessarily care about all of Kubernetes' many built-in resources and their reconciling controllers.
+For more detailed information, check out our [GOALS.md](GOALS.md) doc and our [docs directory](docs/).
 
-#### Embedded/low-resource scenarios?
 
-`kcp` could be _useful_ for environments where resources are scarce, by limiting the number of controllers that need to run. Kubernetes' asynchronous reconciliation pattern can also be very powerful in disconnected or intermittently connected environments, regardless of how workloads actually run.
+## Is `kcp` a "fork" of Kubernetes? 🍴
+
+_No._
 
-### Is that all?
+`kcp` as a prototype currently depends on some unmerged changes to Kubernetes, but we intend to pursue these changes through the usual KEP process, until (hopefully!) Kubernetes can be configured to run as `kcp` runs today.
+
+Our intention is that our experiments _improve Kubernetes for everyone_, by improving [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)s and scaling resource watching, and enabling more, better controllers _for everyone_, whether you're using Kubernetes as a container orchestrator or not.
+
+Our `kcp` specific patches are in the [feature-logical-clusters-1.22](https://github.com/kcp-dev/kubernetes/tree/feature-logical-clusters-1.22) feature branch in the [kcp-dev/kubernetes](https://github.com/kcp-dev/kubernetes) repo. See [DEVELOPMENT.md](DEVELOPMENT.md) for how the patches are structured and how they must be formatted during our experimentation phase.  See [GOALS.md](GOALS.md) for more info on how we intend to use `kcp` as a test-bed for exploring ideas that improve the entire ecosystem.
+
+
+## What's in this repo?
 
-No! See our [GOALS.md](GOALS.md) doc for more on what we are trying to accomplish with this prototype and our [docs/ directory](docs/).
+- **`cmd`**
+    - **`cluster-controller`**
+        * Keeps track of physical `Cluster`s where workloads and other resources are synchronized
+        * Enabled by default in `cmd/kcp`
```

Comment:
wondering how functional that is right now.

Reply:
The cluster controller embedded in kcp?

Reply:
the binary

Reply:
Not 100% sure